spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +71,16 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	@PostConstruct
+	public void deduceModuleArguments() throws Exception {
```

Comment:
> Actually, I think we don't want the concept of label to propagate down to the launcher. That was the rationale behind 2 corresponding arrays. The order is important, and relying on it, no label is necessary.

We need some pivot key of some kind, and the coordinates are not enough (because in `http | filter | f2: filter | log` the expression `org.springframework.cloud.stream.module:filter-processor:1.0.0.BUILD-SNAPSHOT.expression=payload.length()>3` is ambiguous. So I'm not saying that the label "concept" should perspire down here, just that it happens to be a valid pivot key. These keys have no purpose/meaning other that precising which arguments go where